src/OpenTracing/GlobalTracer.php

```diff
@@ -18,7 +18,7 @@
      *
      * @param Tracer $tracer
      */
-    public static function setGlobalTracer(Tracer $tracer)
+    public static function withGlobalTracer(Tracer $tracer)
```

Comment:
I think `set` is better, this is what most PHP libraries do, and it accurately describes what the method does. Any particular reason why you want to avoid it?